javaslang/src/main/java/javaslang/Value.java

```diff
@@ -1032,8 +1035,8 @@ default CharSeq toCharSeq() {
         final Comparator<T> comparator;
         if (this instanceof TreeSet<?>) {
             return (TreeSet<T>) this;
-        } else if (this instanceof SortedSet<?>) {
-            comparator = ((SortedSet<T>) this).comparator();
+        } else if (this instanceof Ordered<?>) {
```

Comment:
here we can't use this code:
```java
if (isOrdered()) {
    Comparator<T> comparator = ((Ordered<T>) this).comparator();
    ...
} else {
   ...
}
```
because `isOrdered()` belongs to `Traversable` but not to `Value`. But `Ordered` combines `PriorityQueue`, `SortedSet` and `SortedMap` and should be useful